java-api-examples/src/DecodeFile.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * // Copyright 2022-2023 by zhaoming
+ */
+/*
+Config modelconfig.cfg
+  #set libsherpa-onnx-jni.so lib root dir
+  solibpath=/sherpa-onnx/build/lib/libsherpa-onnx-jni.so
+  sample_rate=16000
+  feature_dim=80
+  rule1_min_trailing_silence=2.4
+  rule2_min_trailing_silence=1.2
+  rule3_min_utterance_length=20
+  encoder=/sherpa-onnx/build/bin/sherpa-onnx-streaming-zipformer-bilingual-zh-en-2023-02-20/encoder-epoch-99-avg-1.onnx
+  decoder=/sherpa-onnx/build/bin/sherpa-onnx-streaming-zipformer-bilingual-zh-en-2023-02-20/decoder-epoch-99-avg-1.onnx
+  joiner=/sherpa-onnx/build/bin/sherpa-onnx-streaming-zipformer-bilingual-zh-en-2023-02-20/joiner-epoch-99-avg-1.onnx
+  tokens=/sherpa-onnx/build/bin/sherpa-onnx-streaming-zipformer-bilingual-zh-en-2023-02-20/tokens.txt
+  num_threads=4
+  enable_endpoint_detection=false
+  decoding_method=greedy_search
+  max_active_paths=4
+*/
+
+import java.nio.charset.StandardCharsets;

```

Comment:
Could you use some format tools to format your code automatically?

I recommend using google-java-format or clang-format.